go/bindings/combine.go

```diff
@@ -60,6 +60,9 @@ func (c *Combine) Configure(
 	keyPtrs []string,
 	fieldPtrs []string,
 ) error {
+	if len(keyPtrs) == 0 || keyPtrs[0] == "" {
```

Comment:
this will also error on the rust side. I think we remove this and let the Rust error handling stand.

Reply:
removed